### PR TITLE
Changed the logic of displaying widgets on one line

### PR DESCRIPTION
**Pull request by achabalov**
Changed the logic of displaying widgets on one line. The width is set by setting "gridWidth" in view.meta. Maximum width of 24 if we want to fit two widgets on one line put the same value of "position" and the sum of the "gridWidth" of the two widgets should not exceed 24

example

tesler-demo-view

### CODE REVIEW

File: src/main/resources/meta/screen/meeting/view/meetinglist.view.json [hunk 1]
```diff
@@ -11,8 +11,8 @@
     },
     {
       "widgetName": "meetingList",
-      "position": 2,
-      "gridWidth": 4
+      "position": 1,
```
Comment (DRITE) [inline src/main/resources/meta/screen/meeting/view/meetinglist.view.json:14]: @achabalov there is no for changing current layout of desler-demo. It should be `2` as earlier.

File: src/main/resources/db/changelog/0000300_tesler_core.xml [hunk 2]
```diff
@@ -751,7 +751,7 @@
       <column name="DESCRIPTION" type="CLOB"/>
       <column name="SNIPPET" type="CLOB"/>
       <column name="PAGE_LIMIT" remarks="Лимит" type="NUMBER(10, 0)"/>
-      <column defaultValueNumeric="1" name="GRID_WIDTH" type="NUMBER(1, 0)">
+      <column defaultValueNumeric="1" name="GRID_WIDTH" type="NUMBER(2, 0)">
```
Comment (DRITE) [inline src/main/resources/db/changelog/0000300_tesler_core.xml:754]: @achabalov I'm not sure whether it legal. @IgorNB could you check it out please?

**Review by DRITE — CHANGES_REQUESTED** [review]

File: src/main/resources/db/changelog/0000300_tesler_core.xml (hunk 2 above)
Comment (IgorNB) [reply src/main/resources/db/changelog/0000300_tesler_core.xml:754]: It's ok

**Comment by DRITE** [comment]
@IgorNB 

## Instead of migration guide

The feature is a workaround for `@tesler-ui/core` bug which will be fixed in next major release.
Applying the feature to existing project you should pay attention to the value of `gridWidth` in all of presented `*.view.json` files due to it make sense now.
The combination of `position` and `gridWidth` allows you control positioning of widgets on a view. Widgets of the same `position` places on the same row. On the other hand `gridWidth` is responsible for the width of the widget.
`gridWidth` takes values in range between 1 and 24. In case the sum of `gridWidth` values of widgets with the same `position` is larger 24 then widgets from the right will be moved to next row.

For instance:
<table>
<tr><td align="center">6 columns</td><td align="center">6 columns</td><td align="center">6 columns</td><td align="center">6 columns</td></tr>
<tr><td colspan="4" align="center">position: 1<br>gridWidth: 24</td></tr>
<tr><td  colspan="2" align="center">position: 2<br>gridWidth: 12</td><td colspan="2" align="center">position: 2<br>gridWidth: 12</td></tr>
<tr><td align="center">position: 3<br>gridWidth: 6</td><td colspan="3" align="center">position: 3<br>gridWidth: 18</td></tr>
<tr><td align="center">position: 4<br> gridWidth: 6</td></tr>
<tr><td colspan="4" align="center">position: 4<br>gridWidth: 24</td></tr>
</table>

File: src/main/resources/meta/screen/meeting/view/meetinglist.view.json (hunk 1 above)
Comment (achabalov) [reply src/main/resources/meta/screen/meeting/view/meetinglist.view.json]: @DRITE Of course I corrected

**Pull request merged**